detect_os

```diff
@@ -60,8 +60,10 @@ install_deps()
         if ! command -v wget > /dev/null 2>&1; then
             if command -v yum > /dev/null 2>&1; then
                 yum install -y wget
-            else
+            elif command -v apt > /dev/null 2>&1; then
                 apt install -y wget
+            else
+		zypper install -y wget
```

Comment:
Fix indent

Reply:
The file seems to have some inconsistent indents, some tabs, some spaces, some mixed.  In vim it's actually lined up properly.  I'll just cut-n-paste the apt line to get the right indentation but someone else can go fix anything else that's wonky.